src/config/sample-data.js

```diff
@@ -0,0 +1,336 @@
+export const sampleData = {
+  appSessions: [{
+    day: '2021-08-24',
```

Comment:
@hcarneiro It'll be good to make all the dates in this file relative to the date that it's rendered in, e.g. https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/32cfa8d58ce0c3ca00273b047b84990f4d5306c9/js/default-table-data/layouts-table-config.js#L55-L69